app/src/pages/inside/testCaseLibraryPage/testCaseFolders/createFolderModal/createFolderModal.tsx

```diff
@@ -14,23 +14,29 @@
  * limitations under the License.
  */
 
+import { HtmlHTMLAttributes } from 'react';
 import { useIntl } from 'react-intl';
 import { reduxForm, InjectedFormProps } from 'redux-form';
 import classNames from 'classnames/bind';
-import { Modal } from '@reportportal/ui-kit';
+import { FieldLabel, Modal } from '@reportportal/ui-kit';
 
 import { withModal } from 'controllers/modal';
+import { FolderWithFullPath } from 'controllers/testCase';
 import { LoadingSubmitButton } from 'components/loadingSubmitButton';
 import { ModalLoadingOverlay } from 'components/modalLoadingOverlay';
 import { commonValidators } from 'common/utils/validation';
 
 import { commonMessages } from '../../commonMessages';
 import { useFolderModal } from '../shared/useFolderModal';
-import { FolderNameField, ParentFolderToggle, ParentFolderField } from '../shared/FolderFormFields';
+import { FolderNameField, ParentFolderToggle } from '../shared/FolderFormFields';
 import { sharedFolderMessages } from '../shared/sharedMessages';
-import { CREATE_FORM_NAME, PARENT_FIELD_NAME } from '../shared/commonConstants';
+import { CREATE_FORM_NAME, PARENT_FOLDER_FIELD } from '../shared/commonConstants';
 import { FolderFormValues } from '../shared/types';
 
+import { SingleAutocomplete } from 'componentLibrary/autocompletes/singleAutocomplete';
+
+import { AutocompleteOption } from 'componentLibrary/autocompletes/common/autocompleteOption';
```

Comment:
Please pay attention to the order of imports. Above, you have grouped `controllers/*` imports like this. Let's put them there.

Reply:
updated